website/core/DocsRating.tsx

```diff
@@ -4,6 +4,11 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+declare global {
+  interface Window {
+    ga?: (...args: any[]) => void;
+  }
+}
```

Comment:
```suggestion
declare global {
  interface Window {
    ga?: (...args: any[]) => void;
  }
}
```

We should avoid mocking in non-test env and change the `window?.ga` to be optional, it should be sufficient to fix the problem.

Reply:
Let's try adding `@types/google.analytics` then, it should handle the missing types:
* https://www.npmjs.com/package/@types/google.analytics